package.json

```diff
@@ -28,7 +28,6 @@
     "glob": "7.1.1",
     "init-package-json": "^1.2.0",
     "nopt": "4.0.1",
-    "npm": "^2.10.x",
```

Comment:
@stevengill since we are removing npm here, what is the best way to update the scripts in package.json?